### ISSUE TITLE
Entry is no longer possible from the minute the project ends

### ISSUE THREAD

**Issue by HeinzWuert** [kevinpapst/kimai2#1398]
**Describe the bug**
Current beta: following test with 2 successive projects of the same name:
Project aaa_1 ending at 18:00 today
Project aaa_1 starting at 18:01
After the end at 18:00, no more entries are possible in this project. In the selection for entering times, only the follow-up project is displayed.
But the completed project must still be available for data collection.

Solution: Possibly a marking (in color?), which is the completed project and it must remain available until the invoice is closed!

**Comment by HeinzWuert** [comment]
Suggestion:
Show the project start and end date in the filter for the project
or the note "Project ended on xx.xx.".
This way the user knows which is the right one if the names are the same.
However, the project must still be able to be filled with times after the expiry.

**Comment by kevinpapst** [comment]
> Show the project start and end date in the filter for the project or the note beside the name "Project ended on xx.xx.".

Or you choose a different name if it is a different project?!?

> However, the project must still be able to be filled with times after the expiry.

You should only be able to add entries for the time range when the project was active. If a project ended yesterday, you should be able to add a new record for yesterday with the project, but not for today.

**Comment by HeinzWuert** [comment]
It;s not possible to select the ended project because only the new one is displayed!
Just tested once more in the Beta.

**Comment by kevinpapst** [comment]
in which dialog?

**Comment by HeinzWuert** [comment]
https://demo-branch.kimai.org/de/login
admin
projects:  aaa and bbb

**Comment by kevinpapst** [comment]
I renamed the projects to be able to differentiate them.
What exactly should be wrong here? I don't understand it. Works for me

<img width="1513" alt="Bildschirmfoto 2020-01-20 um 19 51 35" src="https://user-images.githubusercontent.com/533162/72750847-46732e00-3bbe-11ea-9750-dcff3cd2f854.png">


<img width="1235" alt="Bildschirmfoto 2020-01-20 um 19 50 32" src="https://user-images.githubusercontent.com/533162/72750824-2ba0b980-3bbe-11ea-9d9a-9debfebb771c.png">

**Comment by HeinzWuert** [comment]
try to make an entry in My Times for bbb_1

**Comment by kevinpapst** [comment]
Why would be bbb any different than aaa?  😁 
It was already in the previous screenshots, but here once again after renaming the projects:

<img width="924" alt="Bildschirmfoto 2020-01-20 um 20 00 20" src="https://user-images.githubusercontent.com/533162/72751371-c0f07d80-3bbf-11ea-8e0e-f9c312d66d7a.png">
<img width="913" alt="Bildschirmfoto 2020-01-20 um 20 00 43" src="https://user-images.githubusercontent.com/533162/72751372-c0f07d80-3bbf-11ea-9bc5-8ef6e0efa772.png">



Maybe I don't understand your problem, could you rephrase it?

**Comment by HeinzWuert** [comment]
Everything can be seen under projects but
under My Times it is not possible to enter times for bbb_1 because it is no longer displayed in the input window. Only bbb_2 is available here.

![image](https://user-images.githubusercontent.com/45666972/72751699-98b54e80-3bc0-11ea-9a5e-f103421ef286.png)

**Comment by kevinpapst** [comment]
Thats not true, have you seen my screenshots?

**Comment by HeinzWuert** [comment]
Understood and confirmed.
Most users like me are working only by selecting a project, not the customer (is faster).
Than in this special case the project can't anymore selected.
Must be very good described for users!
However, when using same project names a hint which one is the older or finished by color or project date behind the name would be helpful.
Thanks for your patience!

**Comment by kevinpapst** [comment]
Booking times on an ended project probably is hopefully not going to happen too often.
So yes, while this is not improved, the users need to be informed, as its definitely not obvious. 

I have to add a listener, that reloads the project list once the start or end date was changed.

**Comment by HeinzWuert** [comment]
The problem will affect all users who do not use the record button for real-time recording and only enter their data afterwards!
A listener, that reloads the project list once the start or end date was changed is therefore a very good idea = Feature request!

**Comment by kevinpapst** [comment]
Is this still a thing? I am pretty sure that I fixed the reloading part when time/date is changed.

**Comment by HeinzWuert** [comment]
to be closed
